packages/augur-ui/src/modules/common/labels.tsx

```diff
@@ -1089,7 +1089,7 @@ export const PendingLabel = (props: PendingLabelProps) => (
         </ReactTooltip>
       </>
     )}
-    {props.status && props.status === TXEventName.Failure && (
+    {status && status === TXEventName.Failure && (
```

Comment:
You don't need this since you've already destructured the props above:
`status && `

Reply:
You actually didn't need it before either

Reply:
status can be null or undefined

Reply:
but I see your point. I'll clean it up more

Reply:
pushed, updated PR